internal/pkg/flags/flags.go

```diff
@@ -133,6 +136,16 @@ func DefineFlagVar() *FlagVar {
 	flag.DurationVar(&flagVar.ManifestRequeueBusyInterval, "manifest-requeue-busy-interval",
 		DefaultManifestRequeueBusyInterval,
 		"determines the duration a Manifest in Processing state is enqueued for reconciliation.")
+	flag.Float64Var(&flagVar.ManifestRequeueJitterProbability, "manifest-requeue-jitter-probability",
+		DefaultManifestRequeueJitterProbability,
+		"determines the probability that jitter is applied to the requeue interval.")
+	flag.Float64Var(&flagVar.ManifestRequeueJitterPercentage, "manifest-requeue-jitter-percentage",
```

Comment:
This must be limited. I'd argue that the jitter should be between [0...5%] (inclusive). I don't have any formula for the 5% value, it's just my gut feeling - the jitter should be almost "invisible" for the end user. If we allow something like 10% it is almost immediately visible for anyone who takes a look at the "reconciliation started" log entries.

Reply:
makes sense